src/core/ComboBox/ComboBox.tsx

```diff
@@ -53,7 +54,8 @@ export type ComboBoxProps<T> = {
    * @default 'No options found'
    */
   emptyStateMessage?: string;
-} & Omit<CommonProps, 'title'>;
+} & Pick<InputContainerProps, 'status' | 'message'> &
```

Comment:
What about all other props? required, disabled, etc... Or those can work through `inputProps` ?

Reply:
Yes, those should use `inputProps`

Reply:
And what about label? Why not this one?

Reply:
I would say expose ability to work with InpuContainer fully as you would have it in  your code.. Not sure this mixture is not confusing..

Reply:
I would like to maintain a separation between label and input for this component.

The "mixture" exists in other components like LabeledInput. Here I'm just selectively exposing props, which is less confusing imo.

If you want, I can remove the `message` prop to reduce prop clutter even more. The main benefit really is the `status` which is hard to achieve from outside.